connection.go

```diff
@@ -1000,7 +1014,7 @@ func (conn *Connection) Do(req Request) *Future {
 			return fut
 		}
 	}
-	return conn.send(req)
+	return conn.send(req, ignoreStreamId)
```

Comment:
How does this one work? Could we be sure that ignoreStreamId is not a valid stream id and can be used in logical conditions?

Reply:
https://www.tarantool.io/en/doc/latest/dev_guide/internals/box_protocol/#box-protocol-streams

```
IPROTO_STREAM_ID can be any positive 64-bit number, and should be unique for the connection. If IPROTO_STREAM_ID equals zero the server instance will ignore it.
```